app/models/consent_form.rb

```diff
@@ -142,7 +156,15 @@ def form_steps
       (:reason if consent_refused?),
       (:injection if consent_refused? && eligible_for_injection?),
       (:gp if consent_given?),
-      (:address if consent_given?)
+      (:address if consent_given?),
+      (:health_1 if consent_given?),
+      (:health_2 if consent_given?),
+      (:health_3 if consent_given?),
+      (:health_4 if consent_given?),
+      (:health_5 if consent_given?),
+      (:health_6 if consent_given?),
+      (:health_7 if consent_given?),
+      (:health_8 if consent_given?)
```

Comment:
Played with the idea of golfing this down but decided this explicit setup looks fine...

Reply:
I spent about 1 min on it and came to a similar conclusion. :P